src/main/resources/modules/home_health_treatment.json

```
@@ -0,0 +1,362 @@
+          "display": "Development of individualized plan of care (procedure)"
+        }
+      ],
+      "direct_transition": "Coordination_Care_Plan"
+    },
+    "Coordination_Care_Plan": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "711069006",
+          "display": "Coordination of care plan (procedure)"
+        }
+      ],
+      "direct_transition": "End_First_HH_Encounter"
+    },
+    "End_First_HH_Encounter": {
+      "type": "EncounterEnd",
+      "direct_transition": "Follow-up_HH_Encounter"
+    },
+    "Follow-up_HH_Encounter": {
+      "type": "Encounter",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "439708006",
+          "display": "Home visit (procedure)"
+        }
+      ],
+      "direct_transition": "Begin_Visit",
+      "encounter_class": "home"
+    },
+    "Begin_Visit": {
+      "type": "Counter",
+      "attribute": "home_health_days",
+      "action": "decrement",
+      "amount": 1,
+      "direct_transition": "Check_Visit_Type"
+    },
+    "Check_Visit_Type": {
+      "type": "Simple",
+      "distributed_transition": [
+        {
+          "distribution": 0.46,
+          "transition": "Therapy_Visit"
+        },
+        {
+          "distribution": 0.54,
+          "transition": "Nursing_Visit"
+        }
+      ]
+    },
+    "Therapy_Visit": {
+      "type": "Simple",
+      "distributed_transition": [
+        {
+          "distribution": 0.74,
+          "transition": "Physical_Therapy"
+        },
+        {
+          "distribution": 0.18,
+          "transition": "Occupational_Therapy"
+        },
+        {
+          "distribution": 0.08,
+          "transition": "Speech_Language_Therapy"
+        }
+      ]
+    },
+    "Nursing_Visit": {
+      "type": "Simple",
+      "distributed_transition": [
+        {
+          "distribution": 0.85,
+          "transition": "Nursing_Care"
+        },
+        {
+          "distribution": 0.15,
+          "transition": "Aide_Visit"
+        }
+      ]
+    },
+    "Physical_Therapy": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "91251008",
+          "display": "Physical therapy procedure (regime/therapy)"
+        }
+      ],
+      "range": {
+        "high": 60,
+        "low": 45,
+        "unit": "minutes"
+      },
+      "direct_transition": "Check for End"
+    },
+    "Occupational_Therapy": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "84478008",
+          "display": "Occupational therapy (regime/therapy)"
+        }
+      ],
+      "direct_transition": "Check for End"
+    },
+    "Speech_Language_Therapy": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "311555007",
+          "display": "Speech and language therapy regime (regime/therapy"
```

missing close paren here
```suggestion
          "display": "Speech and language therapy regime (regime/therapy)"
```

---

Fixed